powerhome/playbook
# Add Props To React Doc Example Files for Dark Mode

#### Breaking Changes

no

#### Runway Ticket URL

[NUXE-236](https://nitro.powerhrg.com/runway/backlog_items/NUXE-236)

#### How to test this

View the doc examples and ensure no errors occur and all examples are visible and working.

#### Checklist:


*The normal release cut off deadline is 3p EDT each week. Please reach out to the release team if you have an urgent request or need a release off cycle.*
